test/runtests.jl

```diff
@@ -1,13 +1,137 @@
 using OpenAIGym
-using Base.Test
+using PyCall
+using Test
 
-# write your own tests here
-@test 1 == 1
+"""
+`function time_steps(env::GymEnv{T}, num_eps::Int) where T`
 
-if isinteractive()
-    env = GymEnv("CartPole-v0")
-    for i=1:5
-        R = run_episode(()->nothing, env, RandomPolicy())
-        info("Episode $i finished. Total reward: $R")
+run through num_eps eps, recording the time taken for each step and
+how many steps were made. Doesn't time the `reset!` or the first step of each
+episode (since higher chance that it's slower/faster than the rest, and we want
+to compare the average time taken for each step as fairly as possible)
+"""
+function time_steps(env::GymEnv, num_eps::Int)
+    t = 0.0
+    steps = 0
+    for i in 1:num_eps
+        reset!(env)
+        # step!(env, rand(env.actions)) # ignore the first step - it might be slow?
+        t += (@elapsed steps += epstep(env))
+    end
+    steps, t
+end
+
+"""
+Steps through an episode until it's `done`
+assumes env has been `reset!`
+"""
+function epstep(env::GymEnv)
+    steps = 0
+    while !env.done
```

Comment:
I think `env.done` should be changed to `finished(env, s)`,
`env.done` is kind of internal variable, so user should not rely on it.

Reply:
👍 and nice, you got the tests working 🎉